app/jinja_filters.py

```diff
@@ -647,22 +656,39 @@ def map_list_collector_config(
                 }
             )
 
-        rows.append(
-            {
-                "rowItems": [
-                    {
-                        "iconType": icon,
-                        "actions": actions,
-                        "rowTitle": item_name,
-                        "id": list_item.get("list_item_id"),
-                        "rowTitleAttributes": {
-                            "data-qa": f"list-item-{index}-label",
-                            "data-list-item-id": list_item.get("list_item_id"),
-                        },
-                    }
-                ]
-            }
-        )
+        row_items = {
+            "iconType": icon,
+            "actions": actions,
+            "id": list_item.get("list_item_id"),
+            "rowTitleAttributes": row_title_attributes,
+        }
+
+        if answer_title:
+            row_items["valueList"] = [{"text": item_name}]
+
+        row_items["rowTitle"] = answer_title or item_name
+
+        rows.append({"rowItems": [row_items]})
+
+        if related_answers:
+            rows[index - 1]["rowItems"].extend(
+                {
+                    "iconType": None,
+                    "actions": [
+                        {
+                            "text": edit_link_text,
+                            "ariaLabel": edit_link_aria_label_text,
+                            "url": f'{list_item.get("edit_link")}#{answer[2]}',
+                            "attributes": {"data-qa": f"list-item-change-{index}-link"},
+                        }
+                    ],
+                    "valueList": [{"text": answer[1]}],
```

Comment:
We can't just manually get the answer from the answer store and set it to display. This currently won't work for anything that needs formatting such as Percentage etc and translations wouldn't work either as we are displaying the value. 

The related answers to a list collector needs to be handled the same way as a normal answer. It should be reusing the same code as a normal summary. In both cases, we have an answer we want to format and display, the only thing that really changes is the fact that, some of them belong to a list.

Reply:
Related answers now being initialised same as normal answers.